pkg/instrumentation/metrics.go

```diff
@@ -56,28 +73,42 @@ func NewMetrics(reg *prometheus.Registry) *Metrics {
 			Name:      RepositoryConfigsTotal,
 			Help:      "Number of repository configurations",
 		}),
-		PublicRepositoriesNotIntrospectedLast24HoursTotal: promauto.With(reg).NewGauge(prometheus.GaugeOpts{
+		PublicRepositories36HourIntrospectionTotal: *promauto.With(reg).NewGaugeVec(prometheus.GaugeOpts{
 			Namespace: NameSpace,
-			Name:      PublicRepositoriesNotIntrospectedLast24HoursTotal,
+			Name:      PublicRepositories36HourIntrospectionTotal,
 			Help:      "Number of public repositories not introspected into the last 24 hours",
```

Comment:
So on second thought, I wouldn't use this suggestion.

For example:
- `CustomRepositories36HourIntrospectionTotal` has two statuses: `introspected` and `missed`.
- In the case of `introspected` that means the total number of repositories that _have_ attempted introspection in the last 36 hours.
- In the case of `missed` that means the total number of repositories that _have not_ attempted introspection in the last 36 hours (and aren't pending).
- So, despite being from the same metric, depending on the status, the metric kinda has two opposite meanings.

It would probably be better to describe what each status is in the help text to avoid confusion.

This does make me wonder if this should be split into two metrics. But I think having help text that clears up any confusion is good enough.

Reply:
updated the description let me know what you think

Reply:
what about: "Breakdown of public repository count by those that attempted introspection and those that missed introspection.

Reply:
better!